bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -49,14 +49,24 @@ object VoTableParser extends VoTableParser {
   /**
    * parse takes an input stream and attempts to read the xml content and convert it to a VoTable resource
    */
-  def parse(catalog: CatalogName, is: InputStream, checkValidity: Boolean = true): CatalogResult = {
+  def parse(catalog: CatalogName, is: InputStream): CatalogResult = {
     // Load in memory (Could be a problem for large responses)
     val xmlText = Source.fromInputStream(is, "UTF-8").getLines().mkString
 
-    (checkValidity, validate(xmlText)) match {
-      case (true, -\/(e))  => \/.left(ValidationError(catalog))
-      case (false, -\/(e)) => \/.right(parse(XML.loadString(xmlText)))
-      case (_, \/-(r))     => \/.right(parse(XML.loadString(r)))
+    validate(xmlText) match {
+      case -\/(e) if catalog.checkValidity => \/.left(ValidationError(catalog))
+      case -\/(e) if catalog == SIMBAD     =>
```

Comment:
I agree it is not satisfying 😄 . The `checkValidity` is there for the cases of other catalogs that are not compliant, though we don't have that case except for `Simbad` at the moment (At a certain time I think ppmxl wasn't compliant either.

I guess I could remove the flag and return to this in case we find another case of funny standard compliance. Now that I think about I could reorg to skip validation altogether for simbad saving some processing time